templates/ep19/bs/talks/talk.html

```diff
@@ -20,24 +20,24 @@
 <div class="container page" id="talk_page">
     <nav aria-label="breadcrumb">
         <ol class="breadcrumb">
-            <li class='breadcrumb-item'><a href="/">Home</a></li>
-            <li class='breadcrumb-item'><a href='#'>Talks</a></li>
-            <li class='breadcrumb-item active'>{{ talk_as_dict.title }}<li>
+            <li class="breadcrumb-item"><a href="/">Home</a></li>
+            <li class="breadcrumb-item"><a href="/events/sessions/#Talks">Talks</a></li>
```

Comment:
@ Reviewer: This is certainly just a "quick fix". Is there a more elegant way to generate this route?

Reply:
Yes, there's a `{% page_url %}` templatetag for django CMS tak takes reverse_id of the page as an argument, I think in this case it should be `{% page_url 'sessions' %}#Talks`. Please not this will have a slightly different breadcrumb than the automatically generated breadcrumb on the `sessions` page.

Reply:
So, I keep it like that? For now. Right?

Reply:
It's fine to keep it like that for now - please leave a `TODO` comment in the template though so that we have a reminder that this needs fixing :)

Reply:
Fixed.